### DISCUSSION TITLE
On branch main nothing to commit, working tree clean

### DISCUSSION THREAD

**Discussion by 28ralph** [community/community#73424] [New to GitHub]
### Hi everyone,

Hello,

I'm having the message "on branch main nothing to commit, working tree clean" Thanks in advance.

**Comment by airtower-luna** [comment]
That's not an error, it means what it says: Everything's committed. But I guess you're having some kind of problem because you're posting here, if you describe it maybe someone can help you. :slightly_smiling_face:
